docs/sources/articles/configuring.md

```diff
@@ -42,12 +58,18 @@ These options :
 
 ## Ubuntu
 
-After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
-running status using Upstart in this way:
+As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
```

Comment:
I think the ubuntu section will have to be split into <= 14.10 and >= 15.04. Afaik, Ubuntu 15.04 switched to `systemd`, which results in people reporting that their configuration is no longer used.